### DISCUSSION TITLE
加载3dtile报错：Rendering has stopped.Unsupported glTF Extension: KHR_technique_webgl

### DISCUSSION THREAD

**Discussion by adwii** [zouyaoji/vue-cesium#287] [Q&A]
加载3dtile时组件报错
![image](https://user-images.githubusercontent.com/35329730/195258004-526f95ff-8b43-45b8-9146-b31aee480e95.png)
组件在ready方法中已添加gltf相关代码
![image](https://user-images.githubusercontent.com/35329730/195258303-47b8f3be-bcdf-4c0a-9b3f-98984e18785e.png)

**Comment by zouyaoji** [comment]
![image](https://user-images.githubusercontent.com/24430116/195261805-546cbe3f-9bd0-45f6-9386-96ddd3c07242.png)

Cesium1.97 版本应该是废弃了 KHR_technique_webgl。确实想用的话通过vue-cesium引入之前的版本即可。


参见：
https://github.com/CesiumGS/cesium/blob/main/CHANGES.md
